plugins/MauticCrmBundle/EventListener/LeadListSubscriber.php

```diff
@@ -133,7 +133,7 @@ public function onLeadListProcessList(ListPreProcessListEvent $event)
                     }
 
                     if (method_exists($integrationObject, 'getCampaignMembers')) {
-                        if ($integrationObject->getCampaignMembers($campaignId, [])) {
+                        if ($integrationObject->getCampaignMembers($campaignId)) {
```

Comment:
This change is causing this warning on SF segments:
```
Rebuilding contacts for segment 14
PHP Warning:  Missing argument 2 for MauticPlugin\MauticCrmBundle\Integration\SalesforceIntegration::getCampaignMembers(), called in plugins/MauticCrmBundle/EventListener/LeadListSubscriber.php on line 136 and defined in plugins/MauticCrmBundle/Integration/SalesforceIntegration.php on line 1343
0 total contact(s) to be added in batches of 300
0 total contact(s) to be removed in batches of 300
0 contact(s) affected
```

Reply:
But other than this warning the Connectwise functionality works fine 👍

Reply:
Ah because I was planning to move that to another SF related PR. But I'll fix SF and deal with the conflicts there.